chain-signatures/node/src/protocol/triple.rs

```diff
@@ -589,69 +588,16 @@ impl TripleManager {
     }
 
     pub async fn contains_mine(&self, id: TripleId) -> bool {
-        self.triple_storage.contains_mine(id, self.me).await
+        self.triple_storage.contains_by_owner(id, self.me).await
     }
 
     pub async fn contains_used(&self, id: TripleId) -> bool {
         self.triple_storage.contains_used(id).await
     }
 
-    /// Take two unspent triple by theirs id with no way to return it. Only takes
-    /// if both of them are present.
-    /// It is very important to NOT reuse the same triple twice for two different
-    /// protocols.
-    pub async fn take_two(
```

Comment:
Managers seam redundant now. Do you want to get rid of them in the future?

Reply:
so they will just become their own individual tasks in the future. They should solely be responsible for spawning new generators (either through stockpiling or receiving a request to start a new one).